plugins/scaffolder-backend-module-rails/src/actions/fetch/rails/railsNewRunner.ts

```diff
@@ -77,6 +77,11 @@ export class RailsNewRunner {
       if (!imageName) {
         throw new Error('No imageName provided');
       }
+      if (!this.containerRunner) {
```

Comment:
Isn't it a bit confusing that the parameter is marked as optional but actually it is mandatory as it will fail with error if not provided

Reply:
No I think it was more confusing before. It had to be required but it wasn't always used if you had the command installed locally. This is the same pattern as the `cookiecutter` one. That if you have the command installed locally it will use that one.

Reply:
https://github.com/backstage/backstage/blob/blam/scaffolder-modules/plugins/scaffolder-backend-module-rails/src/actions/fetch/rails/railsNewRunner.ts#L61-L76